ovos_dinkum_listener/service.py

```diff
@@ -791,11 +794,14 @@ def _handle_listen(self, message: Message):
                 if not self.config["listener"].get("instant_listen"):
                     self.voice_loop.state = ListeningState.CONFIRMATION
                     self.voice_loop.confirmation_event.clear()
-                    Timer(0.5, lambda: self.voice_loop.confirmation_event.set()).start()
+                    if not self.voice_loop.confirmation_event.wait(0.5):
+                        LOG.warning("Confirmation event timed out!")
+                        self.voice_loop.confirmation_event.set()
                 else:
                     self.voice_loop.state = ListeningState.BEFORE_COMMAND
         else:
             self.voice_loop.state = ListeningState.BEFORE_COMMAND
+        self.voice_loop.reset_speech_timer()
```

Comment:
why was this moved? what are the implications of moving it?

Reply:
This was moved so the timer is reset after the audio playback is completed, so the time it takes to play the confirmation sound isn't factored into the minimum length